lib/handlers.js

```diff
@@ -53,6 +53,27 @@ module.exports = (self) => {
         self.apos.doc.addContextOperation(exportOperation);
         self.apos.doc.addContextOperation(importOperation);
       }
+    },
+    'apostrophe:destroy': {
+      async clearTimers() {
+        const ids = Object.keys(self.timeoutIds);
+        if (!ids.length) {
+          return;
+        }
+        self.apos.util.debug(`Clearing ${ids.length} timer(s)`);
+        for (const key of ids) {
+          const entry = self.timeoutIds[key];
+          delete self.timeoutIds[key];
```

Comment:
It's not going to fire this handler twice, you should be able to skip this, but I appreciate your desire to help the GC along.